main.go

```diff
@@ -65,9 +65,21 @@ func run(cfg config.Config) []error {
 		glg.Get().DisableColor()
 	}
 
-	daemon, err := usecase.New(cfg)
-	if err != nil {
-		return []error{errors.Wrap(err, "failed to instantiate daemon")}
+	// Check if X.509 cert mode:
+	useX509Mode := cfg.X509.Cert != "" && cfg.X509.Key != ""
+	glg.Infof("Garm is starting with X.509 mode[%t] ...", useX509Mode)
```

Comment:
```suggestion
	if useX509Mode {
		glg.Infoln("Garm will connect Athenz server with X.509 client certificate ..."
	} else {
		glg.Infoln("Garm will connect Athenz server with n-token ...")
	}
```
more verbose

Reply:
OK

Reply:
Handled in https://github.com/AthenZ/garm/pull/50/commits/2351b1c883e9f2027efbd843c2d90ff181cb5458